tika-pipes-cli/src/main/java/org/apache/tika/pipes/cli/filesystem/FileSystemFetcherCli.java

```diff
@@ -42,6 +43,8 @@ public class FileSystemFetcherCli {
     private Integer port = TIKA_SERVER_GRPC_DEFAULT_PORT;
     @Parameter(names = {"--fetcher-id"}, description = "What fetcher ID should we use? By default will use filesystem-fetcher")
     private String fetcherId = "filesystem-fetcher";
+    @Parameter(names = {"--handler-type"}, description = "Type of the handler to use. Some options are 'TEXT' and 'HTML'. Default is 'TEXT'")
```

Comment:
Can we add all the possible options here?

As well, in https://github.com/atolio/lumen/pull/4475 you reference `text`, `html`, `xml` in lowercase. Let's be consistent with the enumeration.